images/linux/scripts/helpers/install.sh

```diff
@@ -95,3 +95,81 @@ get_github_package_download_url() {
     fi
     echo $downloadUrl
 }
+
+get_github_package_hash() {
+    local repo_owner=$1
+    local repo_name=$2
+    local file_name=$3
+    local url=$4
+    local version=${5:-"latest"}
+    local prerelease=${6:-false}
+    local delimiter=${7:-'|'}
+    local word_number=${8:-2}
+
+    if [ -z "$file_name" ]; then
+        echo "File name is not specified."
+        exit 1
+    fi
+
+    if [ -n "$url" ]; then
+        release_url="$url"
+    else
+        if [ "$version" == "latest" ]; then
+            release_url="https://api.github.com/repos/${repo_owner}/${repo_name}/releases/latest"
+        else
+            json=$(curl -fsSL "https://api.github.com/repos/${repo_owner}/${repo_name}/releases?per_page=100")
+            tags=$(echo "$json" | jq -r --arg prerelease "$prerelease" '.[] | select(.prerelease == ($prerelease | test("true"; "i"))) | .tag_name')
+            tag=$(echo "$tags" | grep -o "$version")
+            if [[ "$(echo "$tag" | wc -l)" -gt 1 ]]; then
```

Comment:
nit: we should either use ` [ condition ]` or `[[ "$condition" ]]` here I see we are using both overall in the helper